.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 node_modules/
 .swp
+src/junit.xml
```

Comment:
Please add a new line at the end of the file

Reply:
Line added

Reply:
Line added